packages/manager/modules/sms/src/sms/sms/compose/telecom-sms-sms-compose.html

```diff
@@ -142,6 +142,21 @@ <h1 data-translate="sms_sms_compose_title"></h1>
                             </div>
                         </div>
 
+                        <div class="form-group" data-ng-if="!$ctrl.isVirtualNumber()">
+                            <div data-ng-if="$ctrl.displaySenderCustomizationAdvice">
```

Comment:
You can get rid of this `<div>` and apply the `ngIf` directive to the parent element like:

```html
<div
    class="form-group"
    data-ng-if="!$ctrl.isVirtualNumber() && $ctrl.displaySenderCustomizationAdvice">
```